src/modules/library.ts

```diff
@@ -26,303 +33,605 @@ export enum Buttons {
 }
 
 export class Library {
+  public driver: WebDriver;
+  private pressDelayInMillis: number;
+  private retryDelayInMillis: number;
+  private retries: number;
   constructor(
-    rokuIPAddress: string,
-    timeoutInMillis = 0,
-    pressDelayInMillis = 0,
-    public driver: WebDriver = new WebDriver(rokuIPAddress, timeoutInMillis, pressDelayInMillis),
+    public rokuIPAddress: string,
+    private username: string,
+    private password: string,
+    {
+      pressDelayInMillis = 1000,
+      retryDelayInMillis = 1000,
+      retries = 1,
+    }: { pressDelayInMillis?: number; retryDelayInMillis?: number; retries?: number },
   ) {
-    this.driver = driver;
-  }
-
-  /** Closes the session */
-  async close(retries = 3) {
-    await this.driver.quiet(retries);
+    this.rokuIPAddress = rokuIPAddress;
+    this.pressDelayInMillis = pressDelayInMillis;
+    this.driver = new WebDriver(rokuIPAddress, retries);
```

Comment:
Did this get left in accidentally?

Reply:
No, I think it is just poorly worded.

The `Library` still has to spin up the `WebDriver`, which is no longer sending commands via Axios to the WebDriverServer, but instead to the Roku directly.

I'll think on how to better name these and fix that.

Reply:
Renamed `Library` to `RokulRunnings` and `WebDriver` to `Driver`
c871193